x-pack/plugins/actions/server/types.ts

```diff
@@ -55,6 +55,12 @@ export interface ActionResult {
   actionTypeId: string;
   name: string;
   config: Record<string, any>;
+  isPreconfigured: boolean;
+  description?: string;
```

Comment:
We use to have something like `description` before but decided to stick with `name`. Is there something that would cause needing both for pre-configured actions?

Reply:
I found it in the yalm example (like 'description: 'Send a message to the #xyz channel''), but it is replaceable with the connector name property in the UI texts